data/tmpl/utility/content3.html

```diff
@@ -18,7 +18,7 @@
           <button class="button" id="template2">Try Shiretoko Fox</button>
         </div>
         <button class="button reset" id="default">Reset to Default</button>
-        <a href="https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&amp;utm_medium=sidebar&amp;utm_campaign=onboardingv1&amp;utm_content=customization" target="_blank">Explore other themes</a>
+        <a href="https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&utm_medium=referral&utm_campaign=onboardingv1" target="_blank">Explore other themes</a>
```

Comment:
@FrancescoSTL wrong medium

URL should be:

https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&utm_medium=firefox-browser&utm_campaign=onboardingv1

I just mentioned this in a comment in #191